HW5/task26.py

```diff
@@ -0,0 +1,14 @@
+# Задача 26:
+# Напишите программу, которая на вход принимает два числа A и B,
+# и возводит число А в целую степень B с помощью рекурсии.
+
+n = int(input('Введите число А: '))
+m = int(input('Введите число В: '))
+
+def f(a,b):
+    if b == 0:
+        return 1
+    else:
+        return f(a,b-1) * a
```

Comment:
Блок кода `else` не нужен, так как `return` завершает функцию.
```suggestion
    return f(a, b-1) * a
```